workflows/alignment-counts-workflow.nf

```diff
@@ -71,79 +75,78 @@ process short_read_alignment {
 
 // COMPUTE ALIGNMENT STATS FOR ALL STATS
 process sam_to_stats {
-
-    //publishDir "${params.phip_data_dir}/stats"
-    label 'samtools'
-
     input:
-        tuple val(sample_id), path(sam_file)
-    
+    tuple val(sample_id), path(sam_file)
     output:
-        path "${sample_id}.stats"
-    
+    path "${sample_id}.stats"
     shell:
     template "sam_to_stats.sh"
-
 }
 
 
 // COMPUTE COUNTS FOR ALL SAMPLES
 process sam_to_counts {
-    
-    //publishDir "${params.phip_data_dir}/counts/"
-    label 'samtools'
-
+    //label 'samtools'
+    publishDir 'counts'
```

Comment:
Just want to flag this for you to come back to -- probably meant to say `publishDir "${params.output}/counts"`